### PR TITLE
more portability changes

### PR DESCRIPTION
**Pull request by moham96**
play sounds in a more pythonic way(using pygame)
port the project to platforms other than Raspberrypi
add archlinux support
more portability changes
parse gassistant-credentials.info file inside python script
update .gitignore

### CODE REVIEW

File: scripts/service-installer.sh [hunk 1]
```diff
@@ -43,7 +43,7 @@ fi
 
 
 for service in systemd/*.service; do
-	sed "s:/home/pi/GassistPi:${repo_path}:g;s:saved-project-id:${projid}:g;s:saved-model-id:${modelid}:g;s:pi:${GIT_OWNER}:g" "$service" \
+	sed "s:/home/pi/GassistPi:${repo_path}:g;s:saved-project-id:${project_id}:g;s:saved-model-id:${device_model_id}:g;s:pi:${GIT_OWNER}:g" "$service" \
```
Comment (shivasiddharth) [inline scripts/service-installer.sh]: As the service file does not have the model id and project id variables, this may be invalid/not required.

**Comment by moham96** [comment]
@shivasiddharth no actually my USB mic died and so I cannot test pushbutton.py in my raspberry pi (that's why I ported the project to archlinux so I can work with it on my laptop) you can merge my commits and revert the changes that I made to the pushbutton.py(which is only the pygame change), until I can get my hands on a microphone for the pi so I can do more tests, there is also GStreamer and pyalsaaudio in case the pygame doesn't work with pushbutton(which is really odd and probably can be fixed

File: scripts/service-installer.sh (hunk 1 above)
Comment (moham96) [reply scripts/service-installer.sh]: I didn't update the pushbutton.py to parse the args from gassistant-credentials.info (as i said in my other comment i can't test this on my pi at the moment) that's why this line is needed for the gassistpi-push-button.service, once i update pushbutton.py to parse the arguments from gassistant-credentials.info this sed expression can be removed

**Comment by shivasiddharth** [comment]
I have merged the commit, but the main.py and pushbutton.py may need some modifications. Your master seems to be behind the patch branch.

**Comment by moham96** [comment]
yes i probably made a mistake working in the master branch

**Pull request merged**